language/stdlib/modules/vector.move

```diff
@@ -62,6 +62,19 @@ module Vector {
         false
     }
 
+    // Return (true, i) if `e` is in the vector `v` at index `i`.
+    // Otherwise returns (false, 0).
+    public fun index_of<Element>(v: &vector<Element>, e: &Element): (bool, u64) {
```

Comment:
Might be simpler to have this abort  if `e` isn't found (especially given that the one call-site we have does that...)

Reply:
I'm not a fan of `abort`ing here. Although, once we have an `Option` I think this should probably return that. My reasoning here is that if we abort here, we are forcing that everyone who doesn't want an abort to occure if the element isn't there traverse the vector twice (once to determine `contains` and the second to `index_of`).

Reply:
Could you add some new functional tests for this?